cpp-ch/local-engine/Common/CHUtil.cpp

```diff
@@ -556,7 +556,7 @@ std::vector<String> BackendInitializerUtil::wrapDiskPathConfig(
     if (path_prefix.empty() && path_suffix.empty())
         return changed_paths;
     Poco::Util::AbstractConfiguration::Keys disks;
-    std::unordered_set<String> disk_types = {"s3", "hdfs_gluten", "cache"};
+    std::unordered_set<String> disk_types = {"s3_gluten", "hdfs_gluten", "cache"};
```

Comment:
GlutenClickHouseWholeStageTransformerSuite configuration of disk type for UT should be changed

Reply:
Disk type is already set to s3_gluten.
![image](https://github.com/user-attachments/assets/490c85df-ed90-4cbf-9d1e-ca33a29ad0bf)